.github/workflows/release.yaml

```diff
@@ -0,0 +1,569 @@
+        with:
+          toolchain: ${{ env.RUST_VERSION }}
+
+      - uses: haskell-actions/setup@v2
+        with:
+          ghc-version: ${{ env.GHC_VERSION }}
+          cabal-version: ${{ env.CABAL_VERSION }}
+          enable-stack: true
+          stack-version: ${{ env.STACK_VERSION }}
+
+      - name: Install flatbuffers
+        run: |
+          wget https://github.com/google/flatbuffers/releases/download/v${{ env.FLATBUFFERS_VERSION }}/MacIntel.flatc.binary.zip -O MacIntel.flatc.binary.zip
+          unzip MacIntel.flatc.binary.zip -d flatbuffers
+          sudo mv flatbuffers/flatc /usr/local/bin/
+
+      - name: Install protobuf
+        run: |
+          curl -L -o protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v${{ env.PROTOC_VERSION }}/protoc-${{ env.PROTOC_VERSION }}-osx-x86_64.zip
+          unzip protoc.zip
+          sudo mv bin/protoc /usr/local/bin/
+          sudo mv include/* /usr/local/include/
+
+      - name: Install Homebrew Packages
+        run: |
+          brew install lmdb llvm
+
+      - name: Build macOS Package
+        run: |
+          printf "N\n" | ./scripts/distribution/macOS-package/build.sh ${{ needs.validate-preconditions.outputs.version }}
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ env.AWS_ROLE_TO_ASSUME }}
+          role-session-name: ${{ github.job }}Session
+          aws-region: "eu-west-1"
+
+      - name: Publish
+        run: |
+          OUTFILE=$(echo '${{ needs.validate-preconditions.outputs.s3_objects }}' | jq -r '.["${{ github.job }}"]')
+          echo "$OUTFILE"
+#          aws s3 cp ./scripts/distribution/macOS-package/build/packages/concordium-node-${TAG}-unsigned.pkg \
+#            $OUTFILE --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
+
+  node-linux:
+     runs-on: ubuntu-latest
+     environment: release
+     needs: [ build-static-binaries, validate-preconditions ]
+     env:
+       DATA_DIR: './scripts/distribution/ubuntu-packages/template/data'
+     strategy:
+       fail-fast: false
+       matrix:
+         node:
+           - env: stagenet
+             tld: com
+             genesis_path: stagenet/2024-09-12/genesis_data
+             grpc_port: 20500
+             listen_port: 9500
+#           - env: testnet
+#             tld: com
+#             genesis_path: testnet/2022-06-13/genesis_data
+#             grpc_port: 20001
+#             listen_port: 8889
+#             exclude_from_release_type: alpha
+#           - env: mainnet
+#             tld: software
+#             genesis_path: mainnet/2021-06-09
+#             grpc_port: 20000
+#             listen_port: 8888
+#             exclude_from_release_type: alpha
+           - env: flynet
+             tld: com
+             genesis_path: flynet/2023-02-01/genesis_data
+             grpc_port: 20002
+             listen_port: 8890
+#         exclude:
```

Comment:
This is supposed to cover the different release types.

Those did not seem to work for now, so lets skip that